README.md

```diff
@@ -159,6 +159,16 @@ Fragments allow slide elements to come one by one. This is often used in lists t
 - come one by one <fragment/>
 ```
 
+Or, if you find it cleaner, like this:
+
+```markdown
+# Slide
+
+-[+] This
```

Comment:
Not sure about this syntax, as it looks pretty close to the GitHub Flavored Markdown `- [x]` syntax for [task list items](https://github.github.com/gfm/#task-list-items-extension-)

Reply:
Haha, yes—you’ve spotted my inspiration. I hadn’t spotted that `+` was an alternative list item syntax. I like the idea of changing its meaning to indicate a fragment, but you’re right that it might not be practical, as this hacky way of doing replacement needs to be rather selective. Just replacing `+` would make any list item that happened to include a `+` a fragment.

Reply:
Hm. I like the idea of using the rather uncommon "+" to support fragments. This would be the "jekyll-revealjs"-flavor. :) It's a breaking change, though.

Reply:
Apologies for the delay… partly because I wasn’t convinced this was going to be possible. I’ve got a candidate working though—please see https://github.com/mrmanc/jekyll-revealjs/pull/1 for the code. As the comment says, it’s pretty hacky but the payoff is cleaner markdown. Give me the thumbs up if you like it @infotexture and @dploeger and I’ll merge it into this PR (being cautious as I’m not confident you’ll approve 😆)

Reply:
I’ve merged https://github.com/mrmanc/jekyll-revealjs/pull/1 so that you can see my changes here and comment in one place. This means the syntax is now simply starting the line with a `+` for a fragment.

What do you think about these changes @infotexture? Particularly the code around [here](https://github.com/dploeger/jekyll-revealjs/pull/26/files#diff-ca18f341df0ed9d5219742abdaf43bfaR56)…

Reply:
@dploeger do you have any input here? I’ll probably go ahead and merge this in the morning if there are no objections.